lib/transformations/resolve/resolve.js

```diff
@@ -0,0 +1,46 @@
+// Press ctrl+space for code completion
+export default function transformer(file, api) {
```

Comment:
I am not yet so much deep into de code and the logic, and is quite hard to understand what is this pr about, imho, in order to make the project more approachable for contributors, we should right away add documentation to the functions that we are doing and add some unit tests that help the reviewers who have no context on a PR to understand what they are reviewing